packages/venia-concept/src/RootComponents/Product/Product.js

```diff
@@ -16,41 +15,6 @@ import productQuery from 'src/queries/getProductDetail.graphql';
  * TODO: Replace with a single product query when possible.
  */
 class Product extends Component {
-    static propTypes = {
-        data: shape({
-            productDetail: shape({
-                total_count: number,
-                items: arrayOf(
-                    shape({
-                        id: number,
-                        sku: string.isRequired,
-                        price: shape({
-                            regularPrice: shape({
-                                amount: shape({
-                                    currency: string.isRequired,
-                                    value: number.isRequired
-                                })
-                            }).isRequired
-                        }).isRequired,
-                        image: string,
-                        image_label: string,
-                        media_gallery_entries: arrayOf(
-                            shape({
-                                label: string,
-                                position: number.isRequired,
-                                disabled: bool,
-                                file: string.isRequired
-                            })
-                        ),
-                        description: string,
-                        short_description: string,
-                        canonical_url: string
-                    })
-                ).isRequired
-            }).isRequired
-        })
-    };
-
     addToCart = async (item, quantity) => {
```

Comment:
@jimbo can we remove this function safely? It's not being used by anything in `pwa-studio` but it is technically part of the public API of this component.